modules/aws-eks-teams/main.tf

```
@@ -5,41 +5,32 @@ resource "kubernetes_namespace" "team" {
   for_each = var.application_teams
   metadata {
     name   = each.key
-    labels = each.value["labels"]
+    labels = lookup(each.value, "labels", {})
   }
 }
 
 # ---------------------------------------------------------------------------------------------------------------------
 # Quotas
 # ---------------------------------------------------------------------------------------------------------------------
 resource "kubernetes_resource_quota" "team_compute_quota" {
-  for_each = var.application_teams
+  for_each = local.team_compute_hard_quotas
   metadata {
     name      = "compute-quota"
     namespace = kubernetes_namespace.team[each.key].metadata[0].name
   }
   spec {
-    hard = {
```

[hard](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs/resources/resource_quota#hard) is an optional attribute and I agree we should offer it as such. To do that, I think we can re-write this as:

```suggestion
    hard = try([each.value], [])
```

This would mean that users would need to know how to spec out the `"requests.cpu" = "4000m"`, etc., but all of that control would be left up to them. 

Thoughts @kcaws / @vara-bonthu / @Zvikan ?